### PR TITLE
Pre check in 32773 introduction

### PR DESCRIPTION
**Pull request by brianseek**
## Description
This is a first pass at the introduction page.
@TODO Add routing to button once routing merged in. Remove appointments once mock API merged in. Resolve answers to questions for UX team. Add unit test for intro.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#32773
department-of-veterans-affairs/va.gov-team#32949

## Testing done
Unit tests for Footer and AppointmentsBlock components

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/pre-check-in/pages/Introduction/index.jsx [hunk 1]
```diff
@@ -1,5 +1,158 @@
...
+          </p>
+          <a href="#">
+            Get tips for protecting your personal health information
+          </a>
+        </>
+      ),
+    },
+  ];
+  const appointmentsDateTime = new Date(appointments[0].startTime);
+  const StartButton = () => (
```
Comment (mdewey) [inline src/applications/check-in/pre-check-in/pages/Introduction/index.jsx:75]: We have an action link in the design library that we should be using 

https://design.va.gov/storybook/?path=/docs/components-action-link--primary

File: src/applications/check-in/pre-check-in/pages/Introduction/index.jsx [hunk 2]
```diff
@@ -1,5 +1,158 @@
...
+          talk to someone right away or need emergency care,
+        </p>
+        <ul>
+          <li>
+            Call <a href="tel:911">911</a>,{' '}
+            <span className="vads-u-font-weight--bold">or</span>
+          </li>
+          <li>
+            Call the Veterans Crisis hotline at{' '}
+            <a href="tel:1-800-273-8255">800-273-8255</a> and select 1
```
Comment (mdewey) [inline src/applications/check-in/pre-check-in/pages/Introduction/index.jsx:123]: We also have a telephone component that handles some stuff for phone numbers

https://design.va.gov/storybook/?path=/docs/components-telephone--default

**Review by mdewey — APPROVED** [review]
Should update to use the components, but other than that it looks good. I will still approve to not block you once you use those.

File: src/applications/check-in/pre-check-in/pages/Introduction/index.jsx (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/pre-check-in/pages/Introduction/index.jsx:80]: Updated!

File: src/applications/check-in/pre-check-in/pages/Introduction/index.jsx (hunk 2 above)
Comment (brianseek) [reply src/applications/check-in/pre-check-in/pages/Introduction/index.jsx]: Updated!

**Review by oseasmoran73 — APPROVED** [review]

**Pull request merged**